ui/pages/Project/components/RnaSeqOutliersTable.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Segment, Header } from 'semantic-ui-react'
+
+import { camelcaseToTitlecase } from 'shared/utils/stringUtils'
+import { GeneSearchLink } from 'shared/components/buttons/SearchResultsLink'
+import DataTable from 'shared/components/table/DataTable'
+import FamilyReads from 'shared/components/panel/family/FamilyReads'
+import { COVERAGE_TYPE, JUNCTION_TYPE } from 'shared/components/panel/family/constants'
+import { ButtonLink } from 'shared/components/StyledComponents'
+
+const ScrollableSegment = styled(Segment)`
+  overflow: auto;
+  max-height: 400px;
+`
+
+const RNA_SEQ_SPLICE_NUM_FIELDS = ['zScore', 'pValue', 'deltaPsi']
+const RNA_SEQ_SPLICE_LOC_FIELDS = ['chrom', 'start', 'end', 'strand']
+const RNA_SEQ_SPLICE_DETAIL_FIELDS = ['type', 'readCount', 'rareDiseaseSamplesWithJunction', 'rareDiseaseSamplesTotal']
+
+const RNA_SEQ_SPLICE_COLUMNS = [
+  {
+    name: 'geneId',
+    content: 'Gene Symbol',
+    format: row => (
+      <GeneSearchLink
```

Comment:
The link should not be a gene search, as every single other place in seqr we have a gene name as a link when you click on it it pops up a modal of gene detail, which is what users will expect this to do. We want that behavior here as well, and then have a gene search link next to the gene name in the same column